.github/workflows/flutter_android_cd_ci.yml

```
@@ -120,98 +113,90 @@ jobs:
       - name: Update version
         id: version
         run: |
+          # Determine release type (default to patch if not manual dispatch)
+          RELEASE_TYPE="${{ github.event.inputs.release_type || 'patch' }}"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)